rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/moderationrequest/ModerationRequestController.java

```diff
@@ -478,4 +483,63 @@ private <T extends org.apache.thrift.TBase<T, F>, F extends org.apache.thrift.TF
         setAddition.apply(addition);
         setDeletion.apply(deletion);
     }
+    
+    @PreAuthorize("hasAuthority('WRITE')")
+    @Operation(
+        summary = "Validate Moderation Request",
+        description = "This endpoint validates a user for moderation request.",
+        tags = {"Moderation Requests"})
+    @RequestMapping(value = MODERATION_REQUEST_URL + "/validate", method = RequestMethod.POST)
+    public ResponseEntity<String> validateModerationRequest(
+            @Parameter(description = "Project id.")
+            @RequestParam String entityType,
+            @Parameter(description = "Entity type", example = "Pass entity type like PROJECT/COMPONENT/RELEASE")
+            @RequestParam String entityId,
+            HttpServletRequest request) throws TException{
+
+        try {
+            User user = restControllerHelper.getSw360UserFromAuthentication();
+            Object entity = getEntityByTypeAndId(entityType, entityId, user);
+            if (entity == null) {
+                return ResponseEntity.status(HttpStatus.NOT_FOUND)
+                        .body("Entity not found for the given ID: " + entityId);
+            }
+            boolean isWriteActionAllowed = restControllerHelper.isWriteActionAllowed(entity, user);
+            if (!isWriteActionAllowed) {
+                return ResponseEntity.status(HttpStatus.ACCEPTED).body(
+                        "Validation successful: User is allowed to raise a moderation request.");
+            }
+            return ResponseEntity.ok("User can't create a MR.");
```

Comment:
If the user allowed to perform the write operation, the message should say something like "User allowed to perform write on entity. MR not required."

Current message is bit confusing.

Reply:
Please update the message. It is still confusing.